src/utils/object.ts

```diff
@@ -0,0 +1,22 @@
+import { sort } from "./array";
+import { polishCurrencyObject } from "../constants/PolishCurrencyObject";
+import { CurrencyResponse } from "../services/CurrencyService";
+
+export function addNewPropertyToTableObjectElement(object: [], newProperty: string, newPropertyValue: string) {
+    return object.forEach((objectElement: any) => {
```

Comment:
I think using 'map' will be more readable for returning modified table. Something like this:


table.map( object => ({ ...object, newProperty: newPropertyValue }))


This way, when we see .map method we know from the beginning that there will be returned modified table.

Reply:
Unfortunatelly it's not working correctly Radzio. You need to think about this one.

Reply:
need to see this one then ;s 
```JavaScript
object.map( objectElement => ({ ...objectElement, newProperty: newPropertyValue }))
```

or even you can try to pass full object as a second parameter instead of passing key and value separately. 

```JavaScript
export function addNewPropertyToTableObjectElement(object: [], newProperty: [key:string]: string) {
 object.map( objectElement => ({ ...objectElement, newProperty }))
}
```

and use it like this: 
```JavaScript
addNewPropertyToTableObjectElement(tableAElement.rates, {table: tableAElement.table})
```

Reply:
Unfortunatelly it doesnt work as well. We can refactor it in other task. Ok ?